src/views/pages/Login/index.js

```diff
@@ -91,6 +102,8 @@ class LoginPage extends Component {
                 <div className="row justify-content-center">
                     <div className="col-5">
                         <h2 className="ml-3">Login</h2>
+                        {this.handleLoginResponse()}
+                        {this.state.loginFailed ? <h6 className="ml-3" style={{ color: 'red' }}>Login attempt failed.</h6> : <div />}
```

Comment:
For now this is fine, but it would be pretty helpful if we could take an error payload and maybe get a more helpful message. I see above in the PR we improved error responses from the service, but they're getting weeded out here. We could display the message that comes from the service, but default to this.